### DISCUSSION TITLE
I was wondering why there is no support for boolean type in the `test.each` test?

### DISCUSSION THREAD

**Discussion by andylib93** [vitest-dev/vitest#4437] [Q&A]
In the documentation for [`test.each`](https://vitest.dev/api/#test-each) it says there are certain types to be referenced for the test name. I was simply just wondering why there is no `%b` declaration for the `boolean` type of an expected output for example. It works when I reference it with `%s`, but it kind of does not feel right to me.

But perhaps there is a simple explanation for this? 😊 is it because it gets formatted either way, as the testname needs to be a string? If so, the documentation could be adjusted, to explain this a little bit further?

**Comment by sheremet-va** [comment]
Vitest uses utils.format https://nodejs.org/api/util.html#utilformatformat-args
